docs/antora-playbook.yml

```diff
@@ -7,7 +7,7 @@ content:
   sources:
     - url: https://github.com/boozallen/aissemble.git
       branches:
-        - 5-migrate-documentation
```

Comment:
I: this is why the pages were not publishing - still looking at my working branch... 🤦